registry/config.go

```diff
@@ -32,6 +32,7 @@ const (
 	REGISTRYSERVER = DEFAULT_V2_REGISTRY
 	INDEXSERVER    = DEFAULT_V1_REGISTRY + "/v1/"
 	INDEXNAME      = "docker.io"
+	NOTARYSERVER   = "https://notary.docker.io"
 
 	// INDEXSERVER = "https://registry-stage.hub.docker.com/v1/"
```

Comment:
I'm really bad at British humor, and I consider myself to have dry humor haha my talks are always like crickets over there

> On Jul 22, 2015, at 11:33, David Lawrence <notifications@github.com> wrote:
> 
> In registry/config.go:
> 
> > @@ -32,6 +32,7 @@ const (
> >  	REGISTRYSERVER = DEFAULT_V2_REGISTRY
> >  	INDEXSERVER    = DEFAULT_V1_REGISTRY + "/v1/"
> >  	INDEXNAME      = "docker.io"
> > +	NOTARYSERVER   = "https://notary.docker.io"
> >  
> >  	// INDEXSERVER = "https://registry-stage.hub.docker.com/v1/"
> British humour doesn't seem to translate very well over github comments :-P
> 
> —
> Reply to this email directly or view it on GitHub.
>